Dockerfile

```diff
@@ -0,0 +1,36 @@
+ARG BASEIMAGE=rust:latest
+
+# BUILDER PATTERN
+
+FROM $BASEIMAGE AS builder
+
+WORKDIR /builder
+
+COPY . .
+
+RUN apt-get update
+RUN apt-get -y install libclang-dev libmetis-dev libscotch-dev
+
+ARG C_INCLUDE_PATH="/usr/include/x86_64-linux-musl"
+ARG CPLUS_INCLUDE_PATH="/usr/include/x86_64-linux-musl"
+
+RUN rustup target add x86_64-unknown-linux-musl
+RUN cargo build --workspace --release --bins --target x86_64-unknown-linux-musl --no-default-features
+
+# FINAL IMAGE
+
+FROM alpine
+
+WORKDIR /coupe
+
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/apply-part /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/apply-weight /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/medit2svg /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/mesh-part /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/mesh-refine /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/mesh-reorder /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/part-bench /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/part-info /bin
+COPY --from=builder /builder/target/x86_64-unknown-linux-musl/release/weight-gen /bin
+
+COPY --from=builder /builder/examples/meshes meshes
```

Comment:
I do not know if this is a good idea. It's nice to have example but the image size might increase and README already explains how to share data with the container.

Reply:
I do agree but I fill like it would be easier for neophytes to have already something to play with right after running the container.

Would leaving only one mesh be a good trade-off ?
I don't know how easy it is to find meshes that lead to simple display.

Reply:
> I don't know how easy it is to find meshes that lead to simple display.

Users will bring their own meshes, there is no point in providing stock meshes for them.

Algorithm devs (us) have their own meshes already, or maybe they could ask when needed.

Found this page when searching for "medit mesh":
https://people.math.sc.edu/Burkardt/data/medit/medit.html

Reply:
Well, you guys have a point. I will remove the embedded mesh !